### ISSUE TITLE
Add support to eruby (syntax highlight)

### ISSUE THREAD

**Issue by daviii-lopes** [geany/geany#3928]
I opened a ruby on rails project in Geany and tried to edit an .erb file, but I noticed that there is no syntax highlight support for this type of file, which appears with all white text. 

![image](https://github.com/user-attachments/assets/27ec3b40-09d9-414d-980f-3bc6ed3fc7b7)

**Comment by elextr** [comment]
You can try using the HTML filetype or the ruby filetype to see if either will do enough to be useful simply by setting the `Document->Set Filetype`.  If one of those works ok enough you can make a custom filetype, see the manual.  Otherwise a lexer needs to be added to the upstream [Lexilla](https://www.scintilla.org/Lexilla.html) project.

**Comment by daviii-lopes** [comment]
@elextr Yep, i tried to set filetype to html, which works sufficiently well, thanks!
